server/routes/approvingLicences/handlers/approvalCases.ts

```diff
@@ -27,7 +26,7 @@ export default class ApprovalCaseRoutes {
           name: convertToTitleCase(`${c.nomisRecord.firstName} ${c.nomisRecord.lastName}`.trim()),
           prisonerNumber: c.nomisRecord.prisonerNumber,
           probationPractitioner: c.probationPractitioner,
-          releaseDate: format(new Date(this.selectReleaseDate(c.nomisRecord)), 'dd MMM yyyy'),
```

Comment:
worth adding a test for this?

I quite like the practice of adding a failing test case and then fixing it